examples/ipynb/webgl_example_1.ipynb

```diff
@@ -189,21 +191,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
```

Comment:
It's good practice in general to review the diffs. Not everyone does it though...